src/main/java/ru/l1ttleO/boyfriend/commands/Help.java

```diff
@@ -20,27 +20,51 @@
 
 import java.util.ArrayList;
 import java.util.Comparator;
+import java.util.HashSet;
 import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
 import org.jetbrains.annotations.NotNull;
-import ru.l1ttleO.boyfriend.CommandHandler;
+import ru.l1ttleO.boyfriend.I18n.BotLocale;
+import ru.l1ttleO.boyfriend.commands.util.CommandHandler;
+import ru.l1ttleO.boyfriend.commands.util.CommandReader;
+import ru.l1ttleO.boyfriend.commands.util.Sender;
+import ru.l1ttleO.boyfriend.commands.util.Sender.ConsoleSender;
+import ru.l1ttleO.boyfriend.commands.util.Sender.MessageSender;
+import ru.l1ttleO.boyfriend.settings.GuildSettings;
 
-public class Help extends Command {
+import static ru.l1ttleO.boyfriend.I18n.tl;
+
+public class Help extends Command implements IChatCommand, IConsoleCommand {
 
     public Help() {
-        super("help", "Показывает эту справку");
+        super("help", "?");
+    }
+
+    @Override
+    public void run(final @NotNull MessageReceivedEvent event, final @NotNull CommandReader reader, final @NotNull MessageSender sender) {
+        help(GuildSettings.PREFIX.get(event.getGuild()), sender);
+    }
+
+    @Override
+    public void run(final @NotNull CommandReader reader, final @NotNull ConsoleSender sender) {
+        help("", sender);
     }
 
-    public void run(final @NotNull MessageReceivedEvent event, final @NotNull String @NotNull [] args) {
-        final StringBuilder text = new StringBuilder("Справка по командам:");
-        final ArrayList<Command> commands = new ArrayList<>(CommandHandler.COMMAND_LIST.values());
-        commands.sort(Comparator.comparing(command -> command.name));
+    public static void help(final @NotNull String prefix, final @NotNull Sender sender) {
+        final boolean isConsole = sender instanceof ConsoleSender;
+        final ArrayList<Command> commands = new ArrayList<>(new HashSet<>(
+            (isConsole ? CommandHandler.COMMANDS_CONSOLE : CommandHandler.COMMANDS_CHAT)
+            .keySet().stream().map(name -> CommandHandler.COMMANDS_ALL.get(name)).toList()));
```

Comment:
```suggestion
        final ArrayList<Command> commands = new ArrayList<>(new HashSet<>(
            (isConsole ? CommandHandler.COMMANDS_CONSOLE : CommandHandler.COMMANDS_CHAT)
            .keySet().stream().map(CommandHandler.COMMANDS_ALL::get).toList()));
```
`Lambda can be replaced with method reference` [intellij]